code-challenges.js

```diff
@@ -21,12 +36,42 @@ var testingArray1 = [23, "Heyyyy!", 45, -9, 0, "Yo", false]
 var testingArray2 = [5, "Hola", 43, -34, "greetings", true]
 // Expected output: [ 2, 1, -1 ]
 
+//Write a function that takes an array
+const onlyNumsModThree = (array) => {
+  //Filter through the array to get an array of only numbers
+  //Map through that array to get the result of mod 3 on each element
+  //Return the result of the final array
+  return array.filter(value => typeof value === "number").map(value => value % 3)
+}
 
-
+console.log("#2:")
+console.log(onlyNumsModThree(testingArray1)); // expect: [ 2, 0, -0, 0 ]
+console.log(onlyNumsModThree(testingArray2), "\n"); // expect: [ 2, 1, -1 ]
 
 
 // --------------------3) Create a function that takes in two arrays as arguments returns one array with no duplicate values.
 
 var testingArray3 = [3, 7, "hi", 10, 3, "hello", 4, "hi"]
 var testingArray4 = [7, "hi", 3, 1, "hi", 4, "hello", 4, 7]
 // Expected output: [ 3, 7, "hi", 10, "hello", 4, 1 ]
+
+//Write a function that takes two arrays
+const noDupes = (array1, array2) => {
+  //Create an array to hold the answer
+  let answerArray = []
+  //Merge the argument arrays into a single array
+  let bigArray = array1.concat(array2)
+  //Map over the merged array, pushing the value into the answer array if it's not already there
+  bigArray.map(value => answerArray.includes(value) ? "" : answerArray.push(value))
+  //Return the answerArray
+  return answerArray
+
```

Comment:
I think you could make this work without the additional answerArray. I think your alternate solution would be a good option. This looks fine, just a thought.